process-migration-service/src/main/java/org/kie/processmigration/service/impl/CredentialsServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package org.kie.processmigration.service.impl;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+
+import org.kie.processmigration.model.Credentials;
+import org.kie.processmigration.service.CredentialsService;
+
+@ApplicationScoped
+public class CredentialsServiceImpl implements CredentialsService {
```

Comment:
in my opinion this should be removed as jBPM should not store any user credentials. Instead it should be configured with admin kind of user that can talk to kie servers and that credentials should come from keystore

Reply:
Then we wouldn't know who did what. Is that acceptable?